docker/js

```diff
@@ -11,7 +11,7 @@ RUN mkdir /nodejs && curl http://nodejs.org/dist/v8.9.0/node-v8.9.0-linux-x64.ta
 ENV PATH $PATH:/nodejs/bin
 
 RUN npm i -g eslint eslint-plugin-import@latest eslint-plugin-node@latest eslint-plugin-promise@latest eslint-plugin-standard@latest
-RUN npm i -g puppeteer mqtt webpack@4.29.5 webpack-cli@3.2.3 html-webpack-plugin@3.2.0 --unsafe-perm=true
+RUN npm i -g puppeteer @babel/polyfill webpack@4.29.5 webpack-cli@3.2.3 html-webpack-plugin@3.2.0 --unsafe-perm=true
```

Comment:
I don't get this change, can you explain it briefly for me please ?

Reply:
This lib seems to be required to fix the `Uncaught ReferenceError: regeneratorRuntime is not defined` error coming from webpack (cf https://github.com/kuzzleio/documentation/pull/304/files#diff-e27397eb7218ebe004e4d0fbca6dc4faL9)

But I must admit I am not a webpack expert and if a more elegant/better solution exists, it is very welcome :)

Reply:
Apparently you just have to update your `.babelrc`
https://github.com/babel/babel/issues/5085#issuecomment-363242788

Reply:
We don't have any .babelrc for the snippets but I finally found a way to target a descent browser version from webpack only.
Thank you!